styles/SobreNosotros.module.css

```diff
@@ -1,23 +1,27 @@
 .container_sobrenosotros {
-    @apply w-4/5 m-auto flex items-center my-4; 
+    @apply w-full m-auto; 
 }
 
 .container_text {
-    @apply w-1/2;
+    @apply w-11/12 m-auto mb-12 max-w-5xl rounded-2xl bg-[#125C13] flex flex-wrap items-center sm:w-3/5 sm:flex-nowrap sm:bg-transparent sm:w-4/5;
 }
 
 .container_child {
-    @apply w-4/5 bg-[#125C13] m-auto text-center p-5 text-[#fff] leading-8 rounded-xl;
+    @apply w-full bg-[#125C13] m-auto text-center p-8 text-[#fff] leading-8 rounded-xl;
```

Comment:
I think `#fff` is already available as just `white`

Reference: https://tailwindcss.com/docs/customizing-colors